### PR TITLE
:sparkles: Build the space-framework image

### PR DESCRIPTION
**Pull request by ezrasilvera**
We don't want to use .git in the docker image build instead we export the needed variables when we call make and then 
when building in the image we use the imported file

## Related issue(s)

Fixes #

### CODE REVIEW

**Review by effi-ofer — APPROVED** [review]

**Comment by ezrasilvera** [comment]
/hold
fixing review comments

File: space-framework/entry.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# This is only a place holder once the integration with helm is done this file will be replaced
+set -e
+
+function echoerr() {
+   echo "ERROR: $1" >&2
+}
+
+function run_space_manager() {
+    echo "--< Starting space-manager >--"
+    if ! space_manager -v=${VERBOSITY} ; then
```
[the remark this answers is not in the document]
Comment (ezrasilvera) [reply space-framework/entry.sh:27]: But I have in the image ENV PATH=/home/spacecore/bin:$PATH   did you test it and it didn't  work ?

[the remark this answers is not in the document]
Comment (ezrasilvera) [reply space-framework/entry.sh:27]: The same as above

File: space-framework/Makefile [hunk 2]
```diff
@@ -14,6 +14,27 @@
...
+CORE_PLATFORMS ?= linux/amd64,linux/arm64,linux/ppc64le # todo linux/s390x  ?
+CORE_IMAGE_REPO ?= quay.io/kubestellar/space-framework
+
+GIT_INFO_FILE := git-info.txt
+#GIT_COMMIT := $(shell git rev-parse --short HEAD || echo 'local')
+#IT_DIRTY := $(shell [ $$(git status --porcelain=v2 | wc -l) == 0 ] && echo 'clean' || echo 'dirty')
+#GIT_VERSION := $(shell go mod edit -json | jq '.Require[] | select(.Path == "k8s.io/kubernetes") | .Version' --raw-output)+kcp-$(shell git describe --tags --match='v*' --abbrev=14 "$(GIT_COMMIT)^{commit}" 2>/dev/null || echo v0.0.0-$(GIT_COMMIT))
+
+BUILD_TIME_TAG := $(shell date -u +b%y-%m-%d-%H-%M-%S)
+GIT_TAG = git-${GIT_COMMIT}-${GIT_DIRTY}
```
[the remark this answers is not in the document]
Comment (ezrasilvera) [reply space-framework/Makefile:36]: Interesting point, I used the same thing KS was using

**Pull request merged**